packages/sdk-v2/src/sdk/analytics/mixpanel/api.ts

```diff
@@ -0,0 +1,27 @@
+import Mixpanel from "mixpanel-browser";
+
+export const MixpanelAPI = {
```

Comment:
https://github.com/GoodDollar/GoodDAPP/pull/3886/files/2de46965eb287f858b3d1d07baccaf81a8dbae7f#r1097099785

Reply:
is this a bug?

Reply:
Yes this is design level bug. You’re hiding interface of your wrapper. There’s actually not many arguments we’re using (just props id event) so it doesn’t takes a lot of time to enumerate them all.
If some cases (like with identify) the declaration is even shorter vs with spear operator

Reply:
no a bug is only when something isnt working as intended.